variables.tf

```
@@ -138,6 +144,49 @@ variable "bus_description" {
   default     = null
 }
 
+variable "logging" {
+  description = "The configuration block for the EventBridge bus logging"
+  type = object({
+    include_detail = optional(string)
+    level          = optional(string)
+
+    cloudwatch_logs = optional(object({
+      enabled         = optional(bool, false)
```

this upsets the happy path - most scenarios would want it enabled if they have provided a config, only opting out in certain scenarios. The same fields below should be updated as well 

```suggestion
      enabled         = optional(bool, true)
```